src/database/schema/ChatSchema.ts

```diff
@@ -43,7 +45,8 @@ class ChatSchema implements BaseDbSchema {
     user,
     status,
     isMe,
-    isContinuous
+    isContinuous,
+    attachmentJson
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachmentJson` parameter is not defined in the constructor parameters. It's better to define it explicitly.
```diff
-    isContinuous,
-    attachmentJson
+    isContinuous: boolean,
+    attachmentJson: AttachmentType
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->